### PR TITLE
Restore OMTMultivariateNormal test

### PR DESCRIPTION
**Pull request by neerajprad**
Based on @fehiepsi's diagnosis of the issue in #802.

### CODE REVIEW

File: pyro/distributions/omt_mvn.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 from __future__ import absolute_import, division, print_function
 
 import torch
-from torch.autograd import Function
+from torch.autograd import Function, variable
```
Comment (fehiepsi) [inline pyro/distributions/omt_mvn.py]: @neerajprad is this new style in Pytorch to create variables?

Comment (neerajprad) [reply pyro/distributions/omt_mvn.py]: @fehiepsi - `variable` was introduced sometime back to incorporate support for scalars, e.g. using `variable(1)` will give a 0 dim tensor wrapped inside a `Variable`. In that sense, it is more general than `Variable`. I think soon Variable and Tensor APIs will be fused into a single `tensor` type.

**Comment by neerajprad** [comment]
> great thanks! i would have fixed it myself but i don't want to switch to master quite yet.

Np at all, this was a really small fix. Thanks to @fehiepsi for debugging.
